climpred/exceptions.py

```diff
@@ -0,0 +1,17 @@
+class Error(Exception):
```

Comment:
I'm not familiar with making custom exceptions, but is this necessary?

i.e. can we do `class DimensionError(Exception):`

Reply:
I just followed 8.5 here: https://docs.python.org/3/tutorial/errors.html. I can give it a try without the Error class.

Reply:
I have no idea :D if that's what the docs recommend then I suppose it's the best practice.